src/index.js

```diff
@@ -107,7 +107,8 @@ class Client {
       agentOptions: this.agentOptions,
       baseUrl: `${this.ssl.enabled ? 'https' : 'http'}://${this.host}:${this.port}`,
       strictSSL: this.ssl.strict,
-      timeout: this.timeout
+      timeout: this.timeout,
+      pool: this.pool
```

Comment:
Why not pass as an option?

Reply:
Yeah

Reply:
@Totenfluch Will you fix this? If not, will close the PR.